Source/ParameterEncoding.swift

```diff
@@ -220,9 +220,7 @@ public enum ParameterEncoding {
         let generalDelimitersToEncode = ":#[]@" // does not include "?" or "/" due to RFC 3986 - Section 3.4
         let subDelimitersToEncode = "!$&'()*+,;="
 
-        // rdar://26850776
-        // Crash in Xcode 8 Seed 1 when trying to mutate a CharacterSet with remove
-        var allowedCharacterSet = NSMutableCharacterSet.urlQueryAllowed
+        var allowedCharacterSet = CharacterSet.urlQueryAllowed
         allowedCharacterSet.remove(charactersIn: "\(generalDelimitersToEncode)\(subDelimitersToEncode)")
```

Comment:
We should figure out what characters are different in Apple's `urlQueryAllowed` character set than ours and make sure the logic we have here is exactly what we want. Doesn't need to be in this PR though.